examples/models/sklearn_iris/sklearn_iris.ipynb

```diff
@@ -19,138 +19,9 @@
     "pip install seldon-core\n",
```

Comment:
<p>Section about training locally and wrapping to test with Docker was quite useful, IMO. Very introductory to wrapping models.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/seldon-core/pull/2574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>